hack/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -e
+set -o pipefail
+
+# Deploy csi-driver-shared-resource-operator to the cluster
+#
+# Deployment can be tuned using the following environment variables:
+#
+# - OPERATOR_IMAGE: the image for the operator to deploy
+# - DRIVER_IMAGE: the image for the CSI driver
+# - NODE_REGISTRAR_IMAGE: the image for the csi node registrar
+# - LOG_LEVEL: log level for the operator
+
+rm -rf _deploy
+mkdir -p _deploy
+cp -r manifests/* _deploy/
+
+operatorImage=${OPERATOR_IMAGE:-quay.io/openshift/origin-csi-driver-shared-resource-operator:latest}
+driverImage=${DRIVER_IMAGE:-quay.io/openshift/origin-csi-driver-shared-resource:latest}
```

Comment:
This image does not exist yet.  Where as the old name does exist:

```
gmontero ~ $ docker pull quay.io/openshift/origin-csi-driver-shared-resource:latest
Error response from daemon: unauthorized: access to the requested resource is not authorized
gmontero ~ $ docker pull quay.io/openshift/origin-csi-driver-projected-resource:latest
latest: Pulling from openshift/origin-csi-driver-projected-resource






Digest: sha256:58e1b51f5ad0c8731b2cb11af3b621cd2e0f80b91fb7e0a3227b4e406fd49d5c
Status: Downloaded newer image for quay.io/openshift/origin-csi-driver-projected-resource:latest
quay.io/openshift/origin-csi-driver-projected-resource:latest
gmontero ~ $ 
```

Now, looking at openshift/release, I see where @adambkaplan added the new repo, including *I think* the bits to enable the presence of `quay.io/openshift/origin-csi-driver-shared-resource:latest`, i.e. https://github.com/openshift/release/blob/master/ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml#L31

That was dropped in 5 days ago.  So either the promotion/creation is really slow, or some other enablement is needed.

Taking a peek at https://github.com/openshift/release/pull/20910/files and https://github.com/openshift/release/pull/20974/files I am not immediately seeing where those do something which enables the presence of that image.

So

/hold

and when either @coreydaley or I have cycles, we engage with the test platform team to sort out the presense of the image with the new name.

Reply:
afaik @smarterclayton has to make it public on quay as a final step.

Reply:
yep .... the doc link confirming:  https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#existing-repositories

Reply:
@gabemontero I am able to successfully pull this image now that it is public:
```
cdaley@redhat [~/go/src/github.com/openshift/release/core-services/image-mirroring/openshift] (master)
$ docker pull quay.io/openshift/origin-csi-driver-shared-resource:latest
Trying to pull quay.io/openshift/origin-csi-driver-shared-resource:latest...
Getting image source signatures
Copying blob 5193b0db398b done  
Copying blob a50df8fd88fe done  
Copying blob 1cadda38f72d done  
Copying blob 904d3325f999 done  
Copying blob c5f1ec7b5437 done  
Copying blob 40819342b61f done  
Copying config 08657eaf62 done  
Writing manifest to image destination
Storing signatures
08657eaf62c873bed5b87aeaa66598c513dba0526e2689b084a086ef2f3e21b3
```

Reply:
Clayton pulled the trigger and we can now pull the image

Reply:
sorry didn't refresh and see @coreydaley 's post before I put mine :-)

about to try out the deploy script to confirm I see the same positive results Adam did before going on PTO